js/deribit.js

```diff
@@ -93,7 +93,9 @@ module.exports = class deribit extends Exchange {
             'urls': {
                 'test': 'https://test.deribit.com',
                 'logo': 'https://user-images.githubusercontent.com/1294454/41933112-9e2dd65a-798b-11e8-8440-5bab2959fcb8.jpg',
-                'api': 'https://www.deribit.com',
+                'api': {
+                    'rest': 'https://www.deribit.com',
```

Comment:
When changing the structure of the `api` we also have to change the structure of the `test` field for sandboxes/testnets, since `api` is replaced by `test` in sandbox mode:

- https://docs.ccxt.com/en/latest/manual.html#testnets-and-sandbox-environments
- https://github.com/ccxt/ccxt/blob/master/js/base/Exchange.js#L443